.github/workflows/reusable-unit-split.yml

```diff
@@ -0,0 +1,132 @@
+name: reusable-unit-split
+
+on:
+  workflow_call:
+    inputs:
+      directory:
+        required: true
+        type: string
+      go-version:
```

Comment:
We don't use this as we get the value from go.mod

Reply:
removed.